V2/azure_time_tracker/host.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
what is the purpose of using this host.json file

Reply:
It's a config file created by Azure when you create a new project that contains the configuration options that affect all functions in a function app instance, remember this file will be replaced by the host.json of serverless.